samples/I2S/Output/I2sWavPlayer.cs

```diff
@@ -16,7 +16,7 @@ namespace AudioPlayer
     /// You have to provide pin configuration for I2S communication and a full path to the
     /// WAV file you want to play.
     /// </summary>
-    public class I2sWavPlayer : IDisposable
+    public class I2SWavPlayer : IDisposable
```

Comment:
Why the name change? The casing rules call for `I2s` as it was, not `I2S`...

Reply:
https://github.com/nanoframework/Samples/pull/327

Reply:
No worries! This is nothing major, just trying to have things consistent and tidy. 😉